broker/server.py

```diff
@@ -9,25 +9,30 @@
 import uvicorn
 import logging
 import socket
-import shutil
 import time
 import yaml
 import json
-import sys
 import os
 
+import json
+
 from oedisi.componentframework.system_configuration import WiringDiagram, ComponentStruct
 from oedisi.types.common import ServerReply, HeathCheck
 
-
 logger = logging.getLogger('uvicorn.error')
-
+#logger = logging.getLogger(__name__)
 
 app = FastAPI()
 
 is_kubernetes_env = os.environ['KUBERNETES_SERVICE_NAME'] if 'KUBERNETES_SERVICE_NAME' in os.environ else None
 
 WIRING_DIAGRAM_FILENAME = "system.json"
+WIRING_DIAGRAM : WiringDiagram | None = None
+
+# def test_function(json_file = r"C:/Users/alatif/Documents/GitHub/sgidal-example/scenarios/system.json"):
+#     global WIRING_DIAGRAM
+#     data = json.load(open(json_file, "r"))
+#     WIRING_DIAGRAM = WiringDiagram(**data)
```

Comment:
There's a few comments that we don't need.

Reply:
removed